packages/service-library/src/servicelib/fastapi/client_session.py

```diff
@@ -2,9 +2,12 @@
 from fastapi import FastAPI
 
 
-def setup_client_session(app: FastAPI) -> None:
+def setup_client_session(app: FastAPI, *, max_keepalive_connections: int = 20) -> None:
```

Comment:
maybe put this in the settings as a default and not here. We use that pattern everywhere. That way we can properly tune it form the env vars via ops

Reply:
this is an experiment, and this is used elsewhere. 20 is the default in httpx.
there is an ENV variable with a default already.